src/components/application_manager/test/commands/mobile/register_app_interface_request_test.cc

```diff
@@ -201,7 +205,41 @@ class RegisterAppInterfaceRequestTest
   MockHmiInterfaces mock_hmi_interfaces_;
 };
 
-TEST_F(RegisterAppInterfaceRequestTest, DISABLED_Run_MinimalData_SUCCESS) {
+MATCHER_P2(CheckLockScreenIconUrlNotification, connection_key, url, "") {
+  const bool is_request_type_correct =
+      mobile_apis::RequestType::LOCK_SCREEN_ICON_URL ==
+      (*arg)[am::strings::msg_params][am::strings::request_type].asInt();
+  const bool is_function_id_correct =
+      mobile_apis::FunctionID::OnSystemRequestID ==
+      (*arg)[am::strings::params][am::strings::function_id].asInt();
+  const bool is_mobile_protocol_type_correct =
+      am::commands::CommandImpl::mobile_protocol_type_ ==
+      (*arg)[am::strings::params][am::strings::protocol_type].asInt();
+  const bool is_protocol_version_correct =
+      am::commands::CommandImpl::protocol_version_ ==
+      (*arg)[am::strings::params][am::strings::protocol_version].asInt();
+  const bool is_message_type_correct =
+      mobile_apis::messageType::notification ==
+      (*arg)[am::strings::params][am::strings::message_type].asInt();
+  const bool is_connection_key_correct =
+      connection_key ==
+      (*arg)[am::strings::params][am::strings::connection_key].asUInt();
+  const bool is_url_correct =
+      url == (*arg)[am::strings::msg_params][am::strings::url].asString();
+
+  using namespace helpers;
+  return Compare<bool, EQ, ALL>(true,
+                                is_request_type_correct,
+                                is_function_id_correct,
+                                is_mobile_protocol_type_correct,
+                                is_protocol_version_correct) &&
+         Compare<bool, EQ, ALL>(true,
```

Comment:
@Kozoriz, it will look dirty. And because `Compare` currently support only 5 args in the row as maximum, I used it twice.